engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicProcessService.scala

```diff
@@ -566,7 +569,7 @@ object PeriodicProcessService {
     def pickMostImportantActiveDeployment: Option[DeploymentStatus] = {
       val lastActiveDeploymentStatusForEachSchedule =
         latestDeploymentForEachSchedule(activeDeploymentsStatuses)
-          .sortBy(_.runAt)(Ordering[LocalDateTime])
+          .sortBy(_.runAtWithCreatedAt)
       def first(status: PeriodicProcessDeploymentStatus) =
         lastActiveDeploymentStatusForEachSchedule.find(_.status == status)
 
```

Comment:
maybe:
```
 def last(status: PeriodicProcessDeploymentStatus) =
        lastActiveDeploymentStatusForEachSchedule.findLast(_.status == status)
        ```

Reply:
We are going to face crossCompile problems if we do that :(